### PR TITLE
Css grid layout

### PR DESCRIPTION
**Pull request by rlajous**
## Summary
-Grid layout not responsive
## Screenshot
grid layout
## Trello Card
https://trello.com/c/wAukcmPJ/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: Maquetado/scss/tech.scss [hunk 1]
```diff
@@ -1,53 +1,60 @@
...
 
-.tech {
-  padding: 10px 20px;
-  width: $tech-width;
-  height: $tech-height;
-  background-color: $green;
-  display: flex;
-  flex-direction: column;
+.container {
+
```
Comment (damfinkel) [inline Maquetado/scss/tech.scss]: remove empty line

File: Maquetado/scss/tech.scss [hunk 2]
```diff
@@ -1,53 +1,60 @@
...
+  display: grid;
+  grid-template-columns: 300px 200px 200px;
+  //grid-template-columns: 2fr 1fr 1fr;
```
Comment (damfinkel) [inline Maquetado/scss/tech.scss]: remove comment

File: Maquetado/scss/tech.scss [hunk 3]
```diff
@@ -1,53 +1,60 @@
...
-  margin: 10px 20px;
+  grid-template-areas:
+   "principal side1 side2"
+   "principal side3 side4";
+   margin: 50px 100px;
 
-  .image {
-    width: $tech-logo-size;
-  }
+    .tech {
```
Comment (damfinkel) [inline Maquetado/scss/tech.scss]: indent this at the same level as the parent's properties:

```
.parent {
  property1: ...,
  property2: ...,

  .child {
  
  ...
  }
}
```

Apply this to the rest

File: Maquetado/scss/tech.scss [hunk 4]
```diff
@@ -1,53 +1,60 @@
...
-    right: 4px;
-    content: "";
-    transition: 0.5s ease;
-  }
-}
+        .description {
+          font-size: $text-small;
+          text-align: center;
+          margin: inherit 5px;
+        }
```
Comment (damfinkel) [inline Maquetado/scss/tech.scss]: space between classes

File: Maquetado/scss/tech.scss [hunk 5]
```diff
@@ -1,53 +1,60 @@
...
+        &:first-child {
+            grid-area: principal;
+            justify-self: end;
+            &::before{
```
Comment (damfinkel) [inline Maquetado/scss/tech.scss]: space before {

**Review by damfinkel — CHANGES_REQUESTED** [review]

File: Maquetado/scss/tech.scss (hunk 5 above)
Comment (SKOLZ) [reply Maquetado/scss/tech.scss]: and empty line between properties and nested selectors

File: Maquetado/scss/tech.scss [hunk 6]
```diff
@@ -1,53 +1,60 @@
...
-    transition: 0.5s ease;
-  }
-}
+        .description {
+          font-size: $text-small;
+          text-align: center;
+          margin: inherit 5px;
+        }
+        &:first-child {
+            grid-area: principal;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: wrong indentation here

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by damfinkel — APPROVED** [review]

**Pull request merged**